CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```

Comment:
Can you please add the Code of Conduct to this repo separately ( Don't include in this File ) in this PR